config/config_template.yaml

```diff
@@ -63,4 +65,15 @@ nh: 100 # discard reads with more mappings than the indicated number
 # If 'isomir' and 'mirna' are both in the list, a single table with both types
 # is made.
 mir_list: ['isomir', 'mirna', 'pri-mir']
+
+#### ASCII-STYLE ALIGNMENT PILEUPS PARAMETERS ####
+
+# Dictionary with the list of library names to aggregate when performing the
+# pileups as values and the condition as keys. Library names must match the
+# ones in the samples' table `sample` column. The dictionary keys will be used
+# as the pileup's output directory name.
```

Comment:
Is it really the output _directory_? Will this not be part of the file basename, rather? Fine if not, just want to make sure this is accurate

Reply:
Yes, in the scenario in which the user uses this parameter, the output directory would be `RESULTS/PILEUPS/{key}/`. Should I reformulate the sentence?

Reply:
No, no, that's fine :)